documentation/cadl-structure-guidelines.md

```diff
@@ -86,7 +86,6 @@ All packages defined in this repo would use the `@cadl-api-spec` scope and use a
 Each package should have the following minimum structure:
```

Comment:
Should we call out package.json and package-lock.json are not supposed to check into spec repo?

Reply:
I called that out below. I didn't mention the lock but generally the lock file isn't useful without the package.json file.